worlds/tracker/TrackerClient.py

```diff
@@ -272,6 +281,14 @@ def _set_host_settings(self, host):
         return host['universal_tracker']['player_files_path'], report_type, host['universal_tracker'][
             'hide_excluded_locations']
 
+    def write_empty_yaml(self, game, player_name):
+        host = get_settings()
+        path = os.path.join(host['universal_tracker']['player_files_path'], f'{game}_{player_name}.yaml')
```

Comment:
How does this work with the default value (null) does it put the file in the correct folder?

Reply:
Oh, good catch, it probably doesn't 😅

Let me fix that